modules/dataquery/jsx/react.filterBuilder.js

```diff
@@ -50,7 +50,9 @@ var FilterRule = React.createClass({
 				"lessThanEqual" : "<=",
 		    	"greaterThanEqual" : ">=",
 		    	"startsWith" : "startsWith",
-		    	"contains" : "contains"
+		    	"contains" : "contains",
+		    	"isNull" : "isNull",
+		    	"isNotNull" : "isNotNull"
```

Comment:
ping @jstirling91 any progress on this? I think it's the only thing blocking this PR.